packages/react-storage/src/components/StorageBrowser/displayText/types.ts

```diff
@@ -52,6 +52,26 @@ export interface DefaultLocationDetailViewDisplayText
   tableColumnTypeHeader: string;
 }
 
+/**
+ * Action view display text values
+ */
+
+export interface DefaultCreateFolderViewDisplayText
+  extends DefaultActionViewDisplayText {
+  getValidationMessage: (error: Error) => string;
```

Comment:
We don't throw an error on validation failures, but do think it makes sense to use a callback here. Maybe we can just provide the folder name string?